### PR TITLE
Cfgv1 Closes #61

### LINKED ISSUE #61: Make a Configuration File
We desperately need a configuration file. Currently we have SQL data, hostnames, and machine IDs all over the place. This is causing numerous development issues when a constant is changed it one place but not the other.

Please look at https://docs.python.org/3.6/library/configparser.html some ideas on how to handle configuration files.

### PR DESCRIPTION
**Pull request by r3cebarnett**
TL;DR
Added a bare config file basically
Added some suggestions
Cleaned up the connection part of errorsql.py

Lemme know if everything looks alright.

### CODE REVIEW

File: Software/scan_key/config.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[_sql]
+	hostname =
+	database =
+	username =
+	password =
```
Comment (ophilli) [inline Software/scan_key/config.cfg]: Please add a newline here

File: Software/scan_key/ErrorSQL.py [hunk 2]
```diff
@@ -3,11 +3,19 @@
 import pymysql
 import os
 import smtplib
+import configparser
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 
 def errorSQL(id, errMessage):
-    cnx = pymysql.connect(user='CATS', password='CATS', host='192.168.1.2', database='CATS', autocommit=True)
+	config = configparser.RawConfigParser() #instantiate config reader
```
Comment (ophilli) [inline Software/scan_key/ErrorSQL.py:11]: Long term this file should be made a class and these variables should be pulled into `__init__()`.

**Review by ophilli — CHANGES_REQUESTED** [review]
Looks great @r3cebarnett! Please add a new-line to config.cfg though!

File: Software/scan_key/ErrorSQL.py (hunk 2 above)
Comment (r3cebarnett) [reply Software/scan_key/ErrorSQL.py:11]: I agree with this, however I think it should just be made into a new file called sql.py or something and then we can have errorsql.py handle errors and authsql.py handle authentication or something.

Or have send and get methods in sql.py

File: Software/scan_key/config.cfg (hunk 1 above)
Comment (r3cebarnett) [reply Software/scan_key/config.cfg]: Done I think

**Review by r3cebarnett — COMMENTED** [review]
Updated.

**Comment by ophilli** [comment]
@r3cebarnett Thanks for updating.

It's a style convention from C - It also fixes a couple of github "warnings". Check [this](https://www.quora.com/Why-is-it-a-common-practice-to-put-a-new-line-at-the-end-of-a-source-code-file) out for more info.

**Review by gacosta1 — CHANGES_REQUESTED** [review]
@gmross Please review this before Wednesday morning. Feel free to express your thoughts!

**Comment by ophilli** [comment]
@r3cebarnett Check this out - I just had a major bug on my own code with this. http://stackoverflow.com/questions/13800515/cant-load-relative-config-file-using-configparser-from-sub-directory

**Review by gacosta1 — DISMISSED** [review]

**Review by ophilli — DISMISSED** [review]

**Comment by r3cebarnett** [comment]
@ophilli 
do you want me to go back and implement that?

**Comment by ophilli** [comment]
@r3cebarnett That would be best. This will cause a major issue for CATS down the road if we don't fix it now.

**Comment by r3cebarnett** [comment]
@ophilli Should I make a config folder or just keep it like it is?

**Comment by ophilli** [comment]
I'd say leave it how it is - `software/config.cfg` is a lot cleaner than `software/config/config.cfg`

**Review by ophilli — COMMENTED** [review]
Perfect! Thanks for implementing this @r3cebarnett.

**Comment by r3cebarnett** [comment]
No problem! @ophilli

**Review by gmross — APPROVED** [review]

**Comment by ophilli** [comment]
Excellent. Please merge at your earliest convenience @gmross.

**Pull request merged**